examples/time_frequency/time_frequency_erds.py

```diff
@@ -145,43 +145,43 @@ def map_bands(freq):
 
 g = sns.FacetGrid(df, row='band', col='channel', margin_titles=True)
 g.map(sns.lineplot, 'time', 'value', 'condition', n_boot=10)
-for ax in g.axes.ravel():
-    ax.axhline(0, color='black', linestyle='dashed', linewidth=0.5,
-               alpha=0.5)
-    ax.axvline(0, color='black', linestyle='dashed', linewidth=0.5,
-               alpha=0.5)
-    ax.set(ylim=(None, 1.5))
+g.map(plt.axhline, y=0, color='black', linestyle='dashed', linewidth=0.5,
+      alpha=0.5)
+g.map(plt.axvline, x=0, color='black', linestyle='dashed', linewidth=0.5,
+      alpha=0.5)
+g.set(ylim=(None, 1.5))
 g.set_axis_labels("Time (s)", "ERDS (%)")
 g.set_titles(col_template="{col_name}", row_template="{row_name}")
 g.add_legend()
 
 plt.show()
 
-
 ##############################################################################
 # Having the data in form of a DataFrame also facilitates subsetting,
 # grouping, and other transforms.
-# Here, we use seaborn to plot average ERDS in the motor-imagery interval
+# Here, we use seaborn to plot the average ERDS in the motor-imagery interval
 # as a function of frequency band and imagery condition:
 
-dat = (df.query('time > 1')
-         .groupby(['condition', 'epoch', 'band', 'channel'])[['value']]
-         .mean()
-         .reset_index())
+df_mean = (df.query('time > 1')
+             .groupby(['condition', 'epoch', 'band', 'channel'])[['value']]
+             .mean()
+             .reset_index())
 
-g = sns.FacetGrid(dat, col='condition', col_order=['hands', 'feet'],
+g = sns.FacetGrid(df_mean, col='condition', col_order=['hands', 'feet'],
                   margin_titles=True)
 g = (g.map(sns.violinplot, 'channel', 'value', 'band', n_boot=10,
            palette='deep', order=['C3', 'Cz', 'C4'],
            hue_order=['delta', 'theta', 'alpha', 'beta'])
-      .add_legend(ncol=4, loc=3, bbox_to_anchor=(0.1, 0, 0.9, 0)))
-for ax in g.axes.ravel():
-    ax.axhline(0, color='black', linestyle='dashed', linewidth=0.5,
-               alpha=0.5)
+      .add_legend(ncol=4, loc=3, bbox_to_anchor=(0.1, 0.1, 0.9, 0.1)))
```

Comment:
```suggestion
      .add_legend(ncol=4, loc='lower center'))
```

right now in [the latest rendering](https://27718-1301584-gh.circle-artifacts.com/0/dev/auto_examples/time_frequency/time_frequency_erds.html) the legend does not end up centered in the figure; I think this will fix that.

Reply:
Thx, implemented.